app/models/event.rb

```diff
@@ -0,0 +1,6 @@
+class Event < ActiveRecord::Base
+  belongs_to :tutor
+  has_many :user
+
+  validates :tutor_id, presence: true
```

Comment:
might be better to use `validates_presence_of`. This is new in Rails 4 and also is more readable.